views/backend/postback.php

```diff
@@ -71,39 +10,31 @@
                         ?>
                     </div>
                 </div>
-                <?php
-            }else{
-                $model_article->edit($title,$chapo,$content,$posted, filter_input(INPUT_GET, 'id'));
-             
-                if(!empty($_FILES['image']['name']))
-                {
-                    $model_article->update_img($_FILES['image']['tmp_name'], $extension);
-                    header("Location:/liste-de-tous-les-articles");
-                }
-                else
-                {
-                    header("Location:/liste-de-tous-les-articles");
-                }
-                
-                ?>
-                    <script>
-                        window.location.replace("index.php?page=postback&id=<?= filter_input(INPUT_GET, 'id') ?>");
-                    </script> 
-                <?php
-            }
-        }
-?>
+                <?php } ?>
+            
+<h2>Modifier un article</h2>
+        <div class="row center">
+            <div class="row">
+                <img class="responsive-img" src="/public/img/posts/<?= $post->getImage() ?>" alt="<?= $post->getTitle() ?>"/>
+            </div>
+        </div>
+    <div class="container">
 
     <form method="post" enctype="multipart/form-data">
         <div class="row">
+            
+            <div class="input-field col s12">
+                <div id="content" class="materialize-textarea"><?= $admins->getEmail() ?></div>
+            </div>
+            <hr>
             <div class="input-field col s12">
-                <input type="text" name="title" id="title" value="<?= $post->title ?>"/>
+                <input type="text" name="title" id="title" value="<?= $post->getTitle() ?>"/>
             </div>
             <div class="input-field col s12">
-                <textarea id="content" name="chapo" class="materialize-textarea"><?= $post->chapo ?></textarea>
+                <textarea id="content" name="chapo" class="materialize-textarea"><?= $post->getChapo() ?></textarea>
             </div>
             <div class="input-field col s12">
-                <textarea id="content" name="content" class="materialize-textarea"><?= $post->content ?></textarea>
+                <textarea id="content" name="content" class="materialize-textarea"><?= $post->getContent() ?></textarea>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$post'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)